ensure_aws_default_branch

```diff
@@ -26,17 +24,27 @@ loop do
 end
 
 aws_cc_repos.flatten.each do |repo_name|
+  target_default_branch = "master"
+
+  begin
+    resp = aws_cc_client.get_branch(repository_name: repo_name, branch_name: "main")
+    target_default_branch = "main" if resp.branch.branch_name == "main"
```

Comment:
This `if` condition looks unnecessary to me.

Presumably the `get_branch(..., branch_name: "main")` call is always going to return `branch.branch_name == "main"`?

If that's not guaranteed then I think it would be better to have an error condition like:

```
raise "AWS is being weird" unless resp.branch.branch_name == "main"
target_default_branch = "main"
```

... but probably I'd just remove the conditional.

Reply:
Ah yup! It just raises the error if the branch name doesn't match, probably is safe to remove.